### PR TITLE
Update objects.md

### PR DESCRIPTION
**Pull request by usseoulusa**
greeting() does not console.log and person.greeting() does not produce any output and requires console.log

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/javascript/concepts/objects/objects.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ Person {
 } 
 */
 
-person.greeting();
+console.log(person.greeting());
 // Output: "Hi, I am Elizabeth Harmon"
```
Comment (mohitO12-dev) [inline content/javascript/concepts/objects/objects.md:171]:
```suggestion
// Output: "Hi, I am Elizabeth Harmon"
```

output should look like this but // Output: Hi, I am Elizabeth Harmon.

**Review by mohitO12-dev — COMMENTED** [review]
comments added.

File: content/javascript/concepts/objects/objects.md (hunk 1 above)
Comment (usseoulusa) [reply content/javascript/concepts/objects/objects.md]: I made changes as suggested by removing the double quotes and adding a period at the end.

**Review by mohitO12-dev — COMMENTED** [review]
APPROVED

**Review by Sriparno08 — APPROVED** [review]
Looks good, @usseoulusa! 👍

**Pull request merged**